tools/sense_studio/annotation.py

```diff
@@ -53,7 +55,7 @@ def show_video_list(project, split, label):
 
     video_list = zip(videos, tagged, list(range(len(videos))))
     return render_template('video_list.html', video_list=video_list, split=split, label=label, path=path,
-                           project=project)
+                           project=project, labels=labels)
```

Comment:
Is this still necessary now that we have a context processor?

Reply:
Not need, same for this as video_list.html